go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/gliderlabs/ssh v0.2.2
 	github.com/google/go-containerregistry v0.15.2
 	github.com/ipfs/go-log v1.0.5
-	github.com/kairos-io/kairos-sdk v0.0.7
+	github.com/kairos-io/kairos-sdk v0.0.9
 	github.com/kairos-io/kairos/v2 v2.0.3
```

Comment:
why does this number does not get bumped to v2.3.0?

Reply:
yeah this is a clusterfuck.

Im guessing because the adress this points to is the kairos repo and the libs used to live there up to that point, then we moved all that stuff out of there, to the agent+sdk so its stuck in there.

BUT kairos-agent still is named as `github.com/kairos-io/kairos/v2` when it should move to be named `github.com/kairos-io/kairos-agent/v2` or something.

So there is this disconnect between its old name, the move to the kairos-agent repo and the not-change-of-module-name in the current kairos-agent repo.

PITA

Reply:
I kind of hope that this should fix it but it will require us to update everywhere with the new module: https://github.com/kairos-io/kairos-agent/pull/81

Reply:
cool cool cool